src/main/java/com/bootcamp/demo/service/ScooterService.java

```diff
@@ -0,0 +1,13 @@
+package com.bootcamp.demo.service;
+
+import com.bootcamp.demo.model.Scooter;
+
+import java.util.Set;
+
+public interface ScooterService {
```

Comment:
Not sure if we need the interfaces, but not opposed to having it as such either. Have you discussed as a team which approach would be better? With or without interfaces?

Reply:
As mentioned above, this approach can be debatable.
In this specific scenario what value does having an interface add, when there is only one possible implementations of the Services?